libs/blocks/article-feed/article-helpers.js

```diff
@@ -90,7 +90,11 @@ function loadArticleTaxonomy(article) {
 
     const articleTax = computeTaxonomyFromTopics(topics, path);
 
-    clonedArticle.category = articleTax.category;
+    const { productionDomain, contentRoot } = getConfig();
+
+    if (productionDomain !== 'business.adobe.com' && contentRoot !== '/blog') {
```

Comment:
Since this seems consumer-specific, could it not be sent as a param instead of having a consumer check in Milo?

Reply:
I'd have to look into that, but currently this `article-feed `is tuned for `brand-blog` alone necessitating this change.

The code in this function mutates the original taxonomy data, so maybe there is somewhere farther upstream I could change it. This seemed to be the least intrusive, and now that there is likely another consumer who will be using these blocks, we may want to overhaul this code entirely.

Reply:
@overmyheadandbody based on this discussion: https://github.com/orgs/adobecom/discussions/1507 these files are likely to get highly refactored at a future date to implement any changes to made to the core tagging systems. 

Magma has another requirement that will likely result in a similar addition to these files like the one above: https://jira.corp.adobe.com/browse/MWPW-138963. 

MWPW-138963 and MWPW-138962 are required for the bacom-blog launch happening later this week. I have informed our stakeholders that this will cause tech debt and I will make a ticket based on your observation above. 

@rgclayton do you see any issue adding the above code?

Reply:
@JasonHowellSlavin Just want to point out that the original issue that caused this ticket was initially done on purpose. From 138963 it says

> the titles in our milo consuming repo are pulled from the meta title

That was done so authors can have control and can show a different title in the card feed vs the page itself. Some titles are way too long and needed rephrased. 

This probably should be handled a different way on import instead, and hopefully those intentions aren't lost.

Reply:
@fullcolorcoder this ticket deals with the tag headers on the article cards:

![Screen Shot 2023-11-14 at 3 08 53 PM](https://github.com/adobecom/milo/assets/17607967/2091b91f-a62a-4e0c-b640-11f85a0149f5)

 which seem to default to the first meta `article:tag` on the page. The code in question actually reassigns the category property to a modified version of the taxonomy. In this case, what occurs is `bacom-blog`'s authored pages set the `category` meta property, and this gets ignored on the cards in favor of the first `article:tag`, which is not the desired functionality for our bacom blog. 

> That was done so authors can have control and can show a different title in the card feed vs the page itself. Some titles are way too long and needed rephrased.

This is a great point about https://github.com/adobecom/milo/pull/1528 and I will bring it up with our PMs and Authoring. Essentially, half of the content on `bacom-blog` has the truncated title, half doesn't so maybe it is an authoring retouch. 

This PR, the tags, blocks our launch, the other does not, I don't believe.

Reply:
Resolving, changed the code.